src/telcameras_v2/models.py

```diff
@@ -19,6 +23,7 @@ class Observation(models.Model):
 
 
 class CountAggregate(models.Model):
+    time = TimescaleDateTimeField(interval="1 day", default=timezone.now)
```

Comment:
ik zou dit hernoemen naar `observation_timestamp`, om in lijn te zijn met de `PersonAggregate`